README.md

```diff
@@ -2,11 +2,36 @@
 
 The collection of jobs found in this repository make up the CI Run project.
 CI Runs' aim is, when triggered by a commit to the juju repository is to:
-  - Create s tarball snapshot of the codebase
+  - Create a tarball snapshot of the codebase
   - Build binaries (for use in testing and as agents)
   - Update the testing streams
   - Run the unit tests
-  - Run the suite of functional tests
+  - Run the suite of integration tests
+
+The integration tests themselves are not in this repo, they are in
+`github.com/juju/juju/tests/suites`. This repo contains the informaion Jekins
+needs to run the tests.
+## Changing the tests
+
+To change which clouds and which versions the tests run on, edit
+`./tools/gen-wire-tests/juju.config`. To change which tests are run,
+edit `./jobs/ci-run/integration/integrationtests.yml`.
+
+To build the job descriptions run:
+```
+JUJU_REPO_PATH="<juju-repo-on-branch-to-generate-jobs-from>" make gen-wire-tests
```

Comment:
We have to generate from main atm

Reply:
That is a **very** good point. We should probably get `gen-wire-tests` to check the branch.

Reply:
It already does!
```
JUJU_REPO_PATH="/home/aflynn/Canonical/Juju/juju3/" make gen-wire-tests
2024/10/11 17:13:17 not on the tracking branch "main", or not update to date with HEAD
exit status 1
make: *** [Makefile:61: gen-wire-tests] Error 1
```